### PR TITLE
[#3579] - add usage tracking to webforms

### PR DESCRIPTION
**Pull request by marvinkome**
#### Before the PR (what is the issue or what needed to be done)
Relates #3579 

#### Reviewer Checklist
* [ ] Added an explanation about the work done
* [ ] Connected the PR and the issue on Zenhub
* [ ] Added a test plan to the issue
* [ ] Updated the copyright header (when relevant)
* [ ] Formatted the code
* [ ] Added a documentation (if relevant)
* [ ] Added some unit tests (if relevant)

### CODE REVIEW

File: Dashboard/app/js/lib/components/forms/form-share/index.jsx [hunk 1]
```diff
@@ -18,6 +19,7 @@ export default class WebFormShare extends React.Component {
   copyToClipboard = () => {
     navigator.clipboard.writeText(this.props.data.shareUrl);
     this.setState({ copyToClipboard: true });
+    trackEvent('Webform URL copied');
```
Comment (tangrammer) [inline Dashboard/app/js/lib/components/forms/form-share/index.jsx:22]: don't we want to send any meaningful data inside this event?

Comment (marvinkome) [reply Dashboard/app/js/lib/components/forms/form-share/index.jsx:22]: I can't think of anything else that can/should be tracked, do you have any ideas?

Comment (tangrammer) [reply Dashboard/app/js/lib/components/forms/form-share/index.jsx:22]: @janagombitova  maybe could know better... I'm thinking on formId, or instanceId 🤔

Comment (marvinkome) [reply Dashboard/app/js/lib/components/forms/form-share/index.jsx:22]: I asked her, she said just this text is good enough

Comment (janagombitova) [reply Dashboard/app/js/lib/components/forms/form-share/index.jsx:22]: @tangrammer and @marvinkome would **instanceId** mean the organisation - so icco.akvoflow.org? That would be interesting to see, which partner uses the webforms the most

Comment (marvinkome) [reply Dashboard/app/js/lib/components/forms/form-share/index.jsx:22]: @janagombitova would that mean the current url?

Comment (janagombitova) [reply Dashboard/app/js/lib/components/forms/form-share/index.jsx:22]: I thought about this over the night and decided it is good enough just to have the count. We do not need more for now and should keep things to the minimum. Once we want to know more, we can add the properties later. So no need for extra data

**Pull request merged**